backend/user_management/ut/aws_s3.py

```
@@ -0,0 +1,42 @@
+    region_name=S3_REGION
+)
+
+
+def upload_files_to_s3(files: list[UploadFile], folder: str):
+    """
+    Upload multiple files to AWS S3 and return their URLs.
+    """
+    file_urls = []
+
+    for file in files:
+        file_extension = file.filename.split(".")[-1]
```

Ruff (0.8.2)

Local variable `file_extension` is assigned to but never used

Remove assignment to unused variable `file_extension`

(F841)